internal/plugins/openshift/v1/init_test.go

```diff
@@ -79,9 +79,9 @@ FROM registry.redhat.io/openshift4/ose-ansible-operator:v` + ocpProductVersion +
 FROM quay.io/operator-framework/ansible:latest
 FROM foo/ansible-operator:latest
 
-FROM registry.redhat.io/openshift4/ose-helm-operator:v` + ocpProductVersion + `
-FROM registry.redhat.io/openshift4/ose-helm-operator:v` + ocpProductVersion + `
-FROM registry.redhat.io/openshift4/ose-helm-operator:v` + ocpProductVersion + `
+FROM registry.redhat.io/openshift4/ose-helm-rhel9-operator:v` + ocpProductVersion + `
+FROM registry.redhat.io/openshift4/ose-helm-rhel9-operator:v` + ocpProductVersion + `
```

Comment:
Is there any PR reference to the repo path change to include rhel9? Curious to understand the decision to understand the requirement.

Reply:
@joepvd Can probably provide that context here.

Reply:
For better or worse, it has been a requirement that images on the container catalog have their rhel version in their name. Changing rhel-ness means changing pullspec. (This apparently was not as stictly enforced while images were moving to rhel8)